march_hardware/src/IMotionCube.cpp

```diff
@@ -322,6 +322,13 @@ void IMotionCube::goToOperationEnabled()
   this->goToTargetState(IMotionCubeTargetState::OPERATION_ENABLED);
 }
 
+void IMotionCube::resetIMotionCube()
+{
+  this->setControlWord(0);
+  ROS_DEBUG("Slave: %d, Try to reset IMC", this->slaveIndex);
+  sdo_bit16(slaveIndex, 0x2080, 0, 1);
```

Comment:
You use `this->` in the line above so you should do it here as well. (Actually we use it everywhere)
```suggestion
  sdo_bit16(this->slaveIndex, 0x2080, 0, 1);
```